server/routes/profileRouter.js

```diff
@@ -1,10 +1,12 @@
 const express = require("express");
 const router = express.Router();
 const profileController = require("../controllers/profileController");
+const uploadController = require("../controllers/uploadController");
 
 router.post("/:id", profileController.createProfile);
 router.put("/update:id", profileController.updateProfile);
 router.get("/:id", profileController.findProfile);
 router.get("/all:id", profileController.getAllProfiles);
+router.post("/upload:id", uploadController.setUpload);
```

Comment:
Not sure what your routes look like.

Reply:
updated to.  router.post("/upload", uploadController.setUpload)
there is app.use("/profile" , profileRouter); in the app.js
so the net route become router.post("/profile/upload", uploadController.setUpload)